app/models/miq_expression.rb

```diff
@@ -1101,7 +1100,7 @@ def self.value2human(val, options = {})
       friendly = tables.split(".").collect do|t|
         if t.downcase == "managed"
           val_is_a_tag = true
-          @company + " Tags"
+          "#{User.current_tenant.name} Tags"
```

Comment:
But wouldn't current_tenant be different if the user was in some other tenant under the root tenant?